pages/tasks/index.js

```diff
@@ -57,7 +57,11 @@ const TaskPage = ({ issues }) => {
       </Head>
       <div className="flex w-full items-center justify-center">
         <div className="flex w-full flex-col xl:w-[80%]">
-          <SearchBar searchQuery={searchQuery} setSearchQuery={setSearchQuery} />
+          <SearchBar 
+          searchQuery={searchQuery} 
+          setSearchQuery={setSearchQuery} 
+          placeholder="Search by title, description, or context depth"
```

Comment:
use translation for it

Reply:
@Guboehm , done the translation of this item below